File: Java/OPSJLib/src/ops/Reply.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package ops;

import configlib.ArchiverInOut;
import java.io.IOException;

/**
 *
 * @author Anton Gravestam
 */
public class Reply extends OPSObject
{

    public String requestId = "";
    public boolean requestAccepted = false;
    public String message = "";
    
    @Override
    public void serialize(ArchiverInOut archive) throws IOException
    {
        
        super.serialize(archive);
        requestId = archive.inout("requestId", requestId);
        requestAccepted = archive.inout("requestAccepted", requestAccepted);
        message = archive.inout("message", message);
    }



}

File: C#/Ops/Reply.cs
///////////////////////////////////////////////////////////
//  Reply.cs
//  Implementation of the Class Reply
//  Created on:      12-nov-2011 09:25:35
//  Author:
///////////////////////////////////////////////////////////

namespace Ops 
{
	public class Reply : OPSObject 
    {
		public string message = "";
		public bool requestAccepted = false;
		public string requestId = "";

        public override void Serialize(IArchiverInOut archive)
        {
            base.Serialize(archive);
            requestId = archive.Inout("requestId", requestId);
            requestAccepted = archive.Inout("requestAccepted", requestAccepted);
            message = archive.Inout("message", message);
		}

	}

}